### ISSUE TITLE
[5.x]: Order::EVENT_DEFINE_ADDITIONAL_BUTTONS does nothing

### ISSUE THREAD

**Issue by gopeter** [craftcms/commerce#3692]
### Description

I'm trying to add some buttons to my order like so:

```
        Event::on(
            Order::class,
            Order::EVENT_DEFINE_ADDITIONAL_BUTTONS,
            function (DefineHtmlEvent $event): void {
                $event-html .= "foobar";
            },
        );
```

It works fine on the product page, but not for orders.

### Steps to reproduce

1.  Add something to the HTML via Order::EVENT_DEFINE_ADDITIONAL_BUTTONS

### Expected behavior

That my HTML is appended

### Actual behavior

My HTML is not appended

### Craft CMS version

Craft Pro 5.4.5.1

### Craft Commerce version

5.1.2

### Installed plugins and versions

-

**Comment by nfourtythree** [comment]
Hi @gopeter 

Thank you for reporting this issue.

We have pushed a fix for this which will be included in the next release of Commerce.

To get this early, change your `craftcms/commerce` requirement in your project's `composer.json` to:

```json
"require": {
  "craftcms/commerce": "5.x-dev as 5.1.3",
  "...": "..."
}
```

Then run `composer update`.

As a side, there are some template hooks available to you for the edit order page. Information on those can be found here: https://craftcms.com/docs/commerce/5.x/extend/template-hooks.html#orders

Thanks!

**Comment by lukeholder** [comment]
This is now fixed in [5.1.4](https://github.com/craftcms/commerce/releases/tag/5.1.4) Thanks!
